app/views/layouts/application.html.erb

```diff
@@ -25,6 +25,23 @@
           <li>
             <%= link_to('Timelogs', user_timelogs_path(current_user)) %>        
           </li>
+          <% if Identity.find_by(:provider => "google_oauth2", :user_id => current_user.id) && Identity.find_by(:provider => "trello", :user_id => current_user.id) %>
```

Comment:
too much logic on front end.
I believe we can do some model method and leave front end alone with html and css.
It should look something like this:
`current_user.has_google? current_user.has_trello?